src/main/java/ca/warp7/frc2023/subsystems/Elevator.java

```diff
@@ -0,0 +1,121 @@
+package ca.warp7.frc2023.subsystems;
+
+import static ca.warp7.frc2023.Constants.*;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxAlternateEncoder;
+import com.revrobotics.SparkMaxLimitSwitch;
+import com.revrobotics.SparkMaxLimitSwitch.Type;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Elevator extends SubsystemBase {
```

Comment:
Have important data points logged in shuffleboard

Reply:
@Skii1 if your are not able to do this then explain why